web.Dockerfile

```diff
@@ -1,24 +1,20 @@
 # You can set the Swift version to what you need for your app. Versions can be found here: https://hub.docker.com/_/swift
-FROM swift:4.2 as builder
+FROM swift:5.0 as builder
 
 # For local build, add `--build-arg env=docker`
 # In your application, you can use `Environment.custom(name: "docker")` to check if you're in this env
 ARG env
 
-RUN apt-get -qq update && apt-get -q -y install \
-  tzdata \
-  && rm -r /var/lib/apt/lists/*
 WORKDIR /app
 COPY . .
-RUN mkdir -p /build/lib && cp -R /usr/lib/swift/linux/*.so /build/lib
+RUN mkdir -p /build/lib && cp -R /usr/lib/swift/linux/*.so* /build/lib
 RUN swift build -c release && mv `swift build -c release --show-bin-path` /build/bin
 
 # Production image
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 ARG env
 RUN apt-get -qq update && apt-get install -y \
-  libicu55 libxml2 libbsd0 libcurl3 libatomic1 \
-  tzdata \
```

Comment:
I think we can run into issues with `Foundation.TimeZone` if we don't have this. I'm not sure, though.

Reply:
So should I add tzdata?

Reply:
To the final image? Definitely.

Reply:
I added `DEBIAN_FRONTEND=noninteractive` to the install command to let tzdata get automatically configured with UTC, is that fine?

Reply:
Maybe it would be better to do `ENV DEBIAN_FRONTEND noninteractive`?

That seems to be what other people do: https://github.com/moby/moby/issues/4032

Reply:
Ah - nice reference, but seems like it should be avoided with ENV.